src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -27,29 +28,29 @@ public class SampleDataUtil {
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
-                new Address("Blk 30 Geylang Street 29, #06-40"), new Note("Loves coffee!"),
-                getTagSet("friends"), getAppointmentSet("12:00-13:00 SUN", "00:00-01:00 MON"),
-                getSubjectSet("Math", "Science"), new Level("P1")),
+                new Address("Blk 30 Geylang Street 29, #06-40"), new Note("Week in fractions."),
```

Comment:
Probably should be "weak" instead of "week"

Reply:
Fixed, thanks!